src/content/docs/journey-migration/making-the-switch.mdx

```diff
@@ -19,6 +19,8 @@ Digging deeper into New Relic features will help you do all that and more, and y
 
 ## Do more with your data [#do-more]
 
+In addition to a suite of observability options, New Relic gives you the option to set up and report on custom data, like events and attributes to capture any data that you may want to see. You can setup these data points to be whatever you need them to be to match the needs of your organization. Check out our docs on [custom events and attributes](/docs/data-apis/custom-data/custom-events/report-custom-event-data/) to learn more.
```

Comment:
This paragraph feels pretty abstract. Is there a way we can make it more specific and punchier? The header is a perfect example of where I think you should go with this.

Basically, I think you should go right up to the line of marketing, but not over :)

something like, just spitballing
"We've got a whole suite of observability solutions based on whatever data you're ingesting. But there's more. If you're ingesting custom data, we have tools and resources to help you capture and observe any data that you send us. Check out our docs on... etc"

Or something like that. In general, I think we want to avoid "New Relic" as much as possible.

Reply:
Fixed, thanks!